### PR TITLE
Rebellion - A suitably dark theme.

### PR DESCRIPTION
**Pull request by marjohn56**
There are four files that cause issues with this theme, they are outside of the themes folder and will require modifications.

The first two

gateways.widget.php
status_gateways.php

Have fixed colours and a couple of them are just wrong for this theme.

The other two

jquery.tokenize.css  - Affects the the java handled pages such Intrusion protection

nv.d3.css - Affects nv.d3 objects and the text is black rather than what it needs to be.

### CODE REVIEW

File: themes/rebellion/Makefile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+PLUGIN_NAME=		rebellion
+PLUGIN_VERSION=		1.0
```
Comment (fichtner) [inline themes/rebellion/Makefile]: 0.1

File: themes/rebellion/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (fichtner) [inline themes/rebellion/LICENSE]: no need for this file

we may have to start giving overrides for the license embedded into the package instead

File: themes/rebellion/Makefile [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+PLUGIN_COMMENT=		A suitably dark theme
+PLUGIN_MAINTAINER=	team rebellion
```
Comment (fichtner) [inline themes/rebellion/Makefile]: sorry, must be an email address

File: themes/rebellion/Makefile [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
```
Comment (fichtner) [inline themes/rebellion/Makefile]: add:

PLUGIN_DEVEL= yes

File: themes/rebellion/Makefile [hunk 5]
```diff
@@ -0,0 +1,6 @@
+PLUGIN_NAME=		rebellion
```
Comment (fichtner) [inline themes/rebellion/Makefile]: I'm a bit worried about polluting the plugin namespace. how about "theme-rebellion" ?

**Comment by marjohn56** [comment]
fixed... I think, there are days when I hate  Github.!

File: themes/rebellion/Makefile [hunk 6]
```diff
@@ -0,0 +1,6 @@
+PLUGIN_NAME=		theme-rebellion
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		A suitably dark theme
+PLUGIN_MAINTAINER=	martin@queens-park.com
+PLUGIN_DEVEL - yes
```
Comment (fichtner) [inline themes/rebellion/Makefile]: PLUGIN_DEVEL= yes

:D

Comment (marjohn56) [reply themes/rebellion/Makefile]: You might smile. :-/ give me two

**Comment by marjohn56** [comment]
Done! :)

**Comment by fichtner** [comment]
@marjohn56 one last thing: can we have the path changed from themes/rebellion to misc/theme-rebellion ... it aligns better with the FreeBSD ports

**Comment by marjohn56** [comment]
As I currently have no misc folder, is that/usr/local/misc or user/local/opnsense/misc ?

**Comment by fichtner** [comment]
sorry for not being clearer: for plugins category/pluginname as in plugins.git/misc/theme-rebellion

**Comment by marjohn56** [comment]
33daa5e I think that's what you mean

**Comment by fichtner** [comment]
@marjohn56 yes, will you update the PR?

**Comment by marjohn56** [comment]
I changed my fork and branch as the original was created on the master by mistake... doh.

I will issue a new one,  and reference this one. gimme thirty minutes. :)

**Comment by marjohn56** [comment]
Closed in favour of #494

**Pull request closed without merging**